tests/h/presenters/group_json_test.py

```diff
@@ -81,3 +99,8 @@ def test_asdicts_injects_urls(self, factories):
         for group in result:
             assert group['url']
             assert group['urls']['group']
+
+
+@pytest.fixture
+def group_json_presenter(patch):
```

Comment:
Oh I see, yeah that's unfortunate! Just `GroupJSONPresenter_` with a trailing underscore maybe, sucks but that's the typical Python technique for just avoiding a naming conflict

Reply:
_Or_, since I guess the `group_json_presenter` fixture is only used in the `TestGroupsJSONPresenter` class, if you define the fixture in the test class instead of at module level then it will override the conflicting name, might work